astronomer/airflow/version_check/update_checks.py

```diff
@@ -234,6 +234,20 @@ def available_update(self):
         from .models import AstronomerAvailableVersion
 
         session = get_state(app=current_app).db.session
+        ac_version = version.parse(get_ac_version())
+        main_part = ".".join([str(i) for i in ac_version.release])
+        nearest_release = session.query(AstronomerAvailableVersion).filter(
+            AstronomerAvailableVersion.version.like(f'{main_part}%')
+        ).order_by(AstronomerAvailableVersion.date_released.desc()).first()
```

Comment:
What's preventing `2.2.3-2` from showing when we are already on `2.2.3-2` (this used to happen because of `hidden_from_ui`)?

I wonder if we should still be using `hidden_from_ui` in this query, and that the helper to set it needs to get smarter too?

Reply:
I think it's doing the right thing. `2.2.3-2` is the latest release so no need to show a banner for an update.